src/_includes/site-footer.html

```diff
@@ -0,0 +1,11 @@
+<footer class="site-footer">
+
+    <ul class="u-list-reset">
+        <li class="footer-name">Second Chance</li>
+        <li class="footer-info">1700 Ridgely Street, Baltimore MD 21230</li>
+        <li class="footer-info">Phone: (410)-385-1700 | admin@secondchanceinc.org |
+            donate@secondchanceinc.org | Fax:
```

Comment:
Remember that you can use `a` elements to link email addresses, too:
```suggestion
        <li class="footer-info">Phone: (410)-385-1700 | <a href="mailto:admin@secondchanceinc.org">admin@secondchanceinc.org</a> |
            <a href="mailto:donate@secondchanceinc.org">donate@secondchanceinc.org</a> | Fax:
```